sdk/spring/README.md

```diff
@@ -1,391 +1,116 @@
-azure-cosmosdb-spring-boot-starter | [![Maven Central][cosmosdb-starter-mvn-2.3.x-img]][cosmosdb-starter-mvn-2.3.x] | [![Maven Central][cosmosdb-starter-mvn-2.2.x-img]][cosmosdb-starter-mvn-2.2.x] | [![][cosmosdb-starter-mvn-2.1.x-img]][cosmosdb-starter-mvn-2.1.x] | [![][cosmosdb-starter-mvn-2.0.x-img]][cosmosdb-starter-mvn-2.0.x]
-azure-keyvault-secrets-spring-boot-starter | [![Maven Central][keyvault-starter-mvn-2.3.x-img]][keyvault-starter-mvn-2.3.x] | [![Maven Central][keyvault-starter-mvn-2.2.x-img]][keyvault-starter-mvn-2.2.x] | [![][keyvault-starter-mvn-2.1.x-img]][keyvault-starter-mvn-2.1.x] | [![][keyvault-starter-mvn-2.0.x-img]][keyvault-starter-mvn-2.0.x]
-azure-mediaservices-spring-boot-starter **(Deprecated)** | N/A | [![Maven Central][mediaservices-starter-mvn-2.2.x-img]][mediaservices-starter-mvn-2.2.x] | [![][mediaservices-starter-mvn-2.1.x-img]][mediaservices-starter-mvn-2.1.x] | [![][mediaservices-starter-mvn-2.0.x-img]][mediaservices-starter-mvn-2.0.x]
-azure-servicebus-jms-spring-boot-starter | [![Maven Central][jms-starter-mvn-2.3.x-img]][jms-starter-mvn-2.3.x] | [![Maven Central][jms-starter-mvn-2.2.x-img]][jms-starter-mvn-2.2.x] | [![Maven Central][jms-starter-mvn-2.1.x-img]][jms-starter-mvn-2.1.x] | N/A
-azure-servicebus-spring-boot-starter **(Deprecated)** | N/A | [![Maven Central][servicebus-starter-mvn-2.2.x-img]][servicebus-starter-mvn-2.2.x] | [![][servicebus-starter-mvn-2.1.x-img]][servicebus-starter-mvn-2.1.x] | [![][servicebus-starter-mvn-2.0.x-img]][servicebus-starter-mvn-2.0.x]
-azure-spring-boot-metrics-starter **(Deprecated)** | [![Maven Central][metrics-starter-mvn-2.3.x-img]][metrics-starter-mvn-2.3.x] | [![Maven Central][metrics-starter-mvn-2.2.x-img]][metrics-starter-mvn-2.2.x] | [![][metrics-starter-mvn-2.1.x-img]][metrics-starter-mvn-2.1.x] | [![][metrics-starter-mvn-2.0.x-img]][metrics-starter-mvn-2.0.x]
-azure-storage-spring-boot-starter | N/A | [![Maven Central][storage-starter-mvn-2.2.x-img]][storage-starter-mvn-2.2.x] | [![][storage-starter-mvn-2.1.x-img]][storage-starter-mvn-2.1.x] | [![][storage-starter-mvn-2.0.x-img]][storage-starter-mvn-2.0.x]
+#### Artifact IDs
+
+| Artifact ID                                     |
+| ----------------------------------------------- |
+| azure-spring-boot-starter-active-directory      |
+| azure-spring-boot-starter-active-directory-b2c  |
+| azure-spring-boot-starter-cosmos                |
+| azure-spring-boot-starter-keyvault-certificates |
+| azure-spring-boot-starter-keyvault-secrets      |
+| azure-spring-boot-starter-servicebus-jms        |
+| azure-spring-boot-starter-storage               |
+
+#### Choose Version According to Spring Boot Version
+
+| Spring Boot Version | Azure Spring Boot Version |
+| ------------------- | ------------------------- |
+| 2.5.x               | 3.12.0                    |
```

Comment:
do we need put 3.12.1 here?

Reply:
here does the version point to the bom library or each starter library?

Reply:
Good point.

I didn't found 3.12.1 in the changelog: https://github.com/Azure/azure-sdk-for-java/blob/azure-spring-boot-bom_3.12.1/sdk/spring/azure-spring-boot-starter-active-directory/CHANGELOG.md

I have some suggestions:

1. When release a patch version, release all Spring Cloud Azure artifacts instead of release a specific artifact. 
2. In this PR, just left it to be `3.12.0`.
3. After 3.13.0 released, update the version to 3.13.0

Reply:
that's due to only the bom has a patch version of 3.12.1 but it still manage our starters of 3.12.0.